libcodechecker/analyze/skiplist_handler.py

```diff
@@ -26,27 +26,68 @@ class SkipListHandler(object):
 
     def __init__(self, skip_file):
         """
-        read up the skip file
+        Process the lines of the skip file.
         """
         self.__skip = []
 
-        with open(skip_file, 'r') as skip_file:
-            skip_file_content = [line.strip()
-                                 for line in skip_file
-                                 if line.strip() != '']
+        self.__skip_file_lines = [line.strip() for line
+                                  in skip_file.splitlines()
```

Comment:
Rename this variable to `skip_file_content` and modify the description of the `__init__` function too.